config/resources/configs/frequency-testnet-paseo.json

```diff
@@ -0,0 +1,41 @@
+{
+  "sdkMaxUsersGraphSize": 10000,
+  "sdkMaxStaleFriendshipDays": 90,
+  "maxGraphPageSizeBytes": 1024,
+  "maxPageId": 16,
+  "maxKeyPageSizeBytes": 65536,
+  "graphPublicKeySchemaId": 7,
+  "schemaMap": [
+    [
+      8,
```

Comment:
Just wanted to make sure that these schema id's are the same as what we registered on paseo for each

Reply:
They are. Here's the output of the deploy script:

```
Generated schema mapping:
 {
  "0x203c6838fc78ea3660a2f298a58d859519c72a5efdc0f194abd6f0d5ce1838e0": {
    "tombstone": {
      "1.2": 1
    },
    "broadcast": {
      "1.2": 2
    },
    "reply": {
      "1.2": 3
    },
    "reaction": {
      "1.1": 4
    },
    "profile": {
      "1.2": 5
    },
    "update": {
      "1.2": 6
    },
    "public-key-key-agreement": {
      "1.2": 7
    },
    "public-follows": {
      "1.2": 8
    },
    "private-follows": {
      "1.2": 9
    },
    "private-connections": {
      "1.2": 10
    },
    "public-key-assertion-method": {
      "1.3": 11
    }
  }
}
```